.github/workflows/release.yml

```diff
@@ -188,3 +188,41 @@ jobs:
             ${{ steps.rename_binary.outputs.name }}.tar.gz
             ${{ steps.rename_binary.outputs.name }}.tar.gz.sha256
             assets/Output/*
+
+  publish-winget:
+    needs: release
+    runs-on: macos-14
+    steps:
+      - if: ${{ github.event_name != 'workflow_run' }}
+        uses: vedantmgoyal2009/winget-releaser@v2
+        with:
+          identifier: RussellBanks.Komac
+          installers-regex: 'KomacSetup.*\.exe$'
+          token: ${{ secrets.WINGET_TOKEN }}
+      - if: ${{ github.event_name == 'workflow_run' }}
+        name: Get nightly version
+        run: |
+          $cargo = Invoke-RestMethod -Uri https://raw.githubusercontent.com/russellbanks/Komac/main/Cargo.toml
+          $Version = ($cargo | Select-String -Pattern '(?<=version\s=\s")[\d.]+').Matches.Value
+          "version=$Version" >> $env:GITHUB_OUTPUT
+        id: get-version
+        shell: pwsh
+      - if: ${{ github.event_name == 'workflow_run' }}
+        uses: vedantmgoyal2009/winget-releaser@v2
+        with:
+          identifier: RussellBanks.Komac.Nightly
```

Comment:
I get that this would be a temporary measure but I'd still rather do any nightly releases to winget-pkgs manually because the hash changes very often. If I do that myself, I can hold off making a commit until it's merged in winget-pkgs, for example.